1/src/App.vue

```diff
@@ -16,7 +16,9 @@ export default {
     List,
     Login
   }
+
 };
+
```

Comment:
Try not to clutter "Files Changed" so undo these changes.

Reply:
i updated the pull request...please see last commit